lib/download.sh

```diff
@@ -51,20 +55,30 @@ wget_and_verify()
     fi
     wget "${args[@]}"
 
-    if [[ "${sha256}" =~ https ]]; then
-        sha256="$(curl -SsL "${sha256}")"
+    if [[ "${checksum}" =~ https ]]; then
+        checksum="$(curl -SsL "${checksum}")"
+    fi
+
+    if [[ "${checksum}" =~ ${dlname,,} ]]; then
```

Comment:
This does not work for Tilt. 

1. We download Tilt's installation script, and the checksum is for release tarball. It cannot find install.sh on the checksum file, and skips it.
2. It does not return failure for this case, and happily goes and installs TIlt anyways.

We either need to change the download itself to the release tarball or use hardcoded SHA for the script. Using hardcoded SHA wouldn't be too bad, as it is the same script regardless of arch.

Reply:
Ah, I had missed that we use a different install method for tilt. I can change it back to the pinned SHA for now, but changing the installation method shouldn't be too hard either if we want that instead.